pkgs/by-name/hy/hypercore/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  writeScript,
+}:
+
+buildNpmPackage (finalAttrs: {
+  pname = "hypercore";
+  version = "11.6.2";
+
+  src = fetchFromGitHub {
+    owner = "holepunchto";
+    repo = "hypercore";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-o4Ojja+8Yvn1fK9FszcH/wHZnuyhpHxsch0D1kb5HlE=";
+  };
+
+  npmDepsHash = "sha256-lBV5IehqP2qOe3/mJjLJt8Ty77QN8uUGc5oolFUEqck=";
+
+  dontNpmBuild = true;
+
+  passthru.updateScript = writeScript "update-hypercore" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p  common-updater-scripts curl
+
+    set -eu -o pipefail
+
+    version="$(curl -s https://api.github.com/repos/holepunchto/hypercore/tags | jq -r '.[0].name' | sed 's/^v//')"
+
+    update-source-version hypercore "$version"
+  '';
+
+  meta = {
+    description = "Hypercore is a secure, distributed append-only log.";
```

According to [the docs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes), meta.description should:
- Not start with the package name.
- Not end with a period (or any punctuation for that matter).

```suggestion
    description = "Secure, distributed append-only log";
```